.markdownlint.json

```diff
@@ -67,7 +67,9 @@
       "h1",
       "p",
       "details",
-      "summary"
+      "summary",
+      "Tabs",
+      "TabItem"
```

Comment:
Huh, I wonder if this is a markdownlint bug? That it's picking up on imported words from mdx-style imports?

(not requesting any changes, just... amused.)

Reply:
No, it's because of the actual `<Tabs>` tags. I think it's fine that markdownlint is not making a distinction between upper- and lowercase tag names

Reply:
I didn't know docasaurus would let you embed HTML like that.
our rule docs have been doing tabs like this:

https://github.com/typescript-eslint/typescript-eslint/blob/04baac8d8958a21d9ae3312206dc283fec642887/packages/eslint-plugin/docs/rules/ban-ts-comment.md#L46-L70

Reply:
Ah, it's the feature of MDX. https://docusaurus.io/docs/markdown-features/react

Using comments is a little more indirect, but plays well with different interfaces. I'm not sure which you prefer?

Reply:
Should we change the file extensions to `.mdx`, then?

Reply:
There's no difference between the two, so it's your choice... `.mdx` requires one extra click to set up the editor association and/or install an extra extension